src/hooks/useBasisForm/validation/validateCheckboxGroup.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  CheckboxOptions,
+  InternalCheckboxGroupProps,
+} from "../../../components/CheckboxGroup/types";
```

Comment:
Do you have relative imports setup ? too many `../...`

Reply:
Basis does not have absolute imports, I would have to run the change past misha before making that